File: Web/RMA-Web-Shell/projects/digicare/src/app/medical-form-manager/models/first-medical-report-form.ts
import { MedicalReportForm } from 'projects/digicare/src/app/medical-form-manager/models/medical-report-form';

export class FirstMedicalReportForm {
    firstMedicalReportFormId: number;
    medicalReportForm: MedicalReportForm;
    clinicalDescription: string;
    mechanismOfInjury: string;
    isInjuryMechanismConsistent: boolean;
    isPreExistingConditions: boolean;
    preExistingConditions: string;
    firstDayOff: Date;
    lastDayOff: Date;
    estimatedDaysOff: number;
    firstConsultationDate: Date;
}

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/FirstMedicalReportForm.cs
namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class FirstMedicalReportForm
    {
        public int FirstMedicalReportFormId { get; set; }
        public int MedicalReportFormId { get; set; }
        public string ClinicalDescription { get; set; }
        public string MechanismOfInjury { get; set; }
        public bool IsInjuryMechanismConsistent { get; set; }
        public bool IsPreExistingConditions { get; set; }
        public string PreExistingConditions { get; set; }
        public System.DateTime? FirstDayOff { get; set; }
        public System.DateTime? LastDayOff { get; set; }
        public int? EstimatedDaysOff { get; set; }
        public System.DateTime? FirstConsultationDate { get; set; }

        public MedicalReportForm MedicalReportForm { get; set; }
    }
}
